openpype/modules/slack/plugins/publish/integrate_slack_api.py

```diff
@@ -0,0 +1,83 @@
+try:
+    from slackclient import SlackClient
+    python2 = True
+except ImportError:
+    python2 = False
+    from slack_sdk import WebClient
+    from slack_sdk.errors import SlackApiError
+
+import pyblish.api
+from openpype.lib.plugin_tools import prepare_template_data
+
+
+class IntegrateSlackAPI(pyblish.api.InstancePlugin):
+    """ Send message notification to a channel.
+
+        Triggers on instances with "slack" family, filled by
+        'collect_slack_family'.
+
+        Expects configured profile in
+        Project settings > Slack > Publish plugins > Notification to Slack
+
+        Message template can contain {} placeholders from anatomyData.
+    """
+    order = pyblish.api.IntegratorOrder + 0.499
+    label = "Integrate Slack Api"
+    families = ["slack"]
+
+    optional = True
+
+    def process(self, instance):
+        message_templ = instance.data["slack_message"]
+
+        fill_pairs = set()
+        for key, value in instance.data["anatomyData"].items():
+            if not isinstance(value, str):
```

Comment:
Why are non string values skipped? It won't be possible to use e.g. `{project[name]}`?

Reply:
Changed it to be more explicit